guides/tutorials/issue-a-credential.md

```diff
@@ -84,20 +84,24 @@ following commands can be used:
 # yarn
```

Comment:
the example script uses node and inbound transports for both ends, but the tutorial seems to be written as if assumes the holder is in a RN environment. Do the snippets match with the tutorial?

Reply:
Ah yeah good catch, it used to be RN for the holder but we cannot run the snippets in tests then. I will rewrite the tutorial for node as issuer and holder.

Reply:
Yeah maybe we can add some hints later on that describes what needs to be changed to make the holder work in RN envirionments